samples/nrf5340/netboot/README.rst

```diff
@@ -7,82 +7,126 @@ nRF5340: Network core bootloader
+
+.. |sample path| replace:: :file:`samples/nrf5340/netboot/`
+
+.. include:: /includes/build_and_run.txt
+See also the following pages:
+
+* :ref:`ug_nrf5340_building` for additional details on building samples on the nRF5340.
+* :ref:`configure_application` for additional details on configuring your application.
+
+See below for an example of a multi-image build that includes :ref:`peripheral_uart`, using either SEGGER Embedded Studio or the command line:
+
+.. tabs::
 
-To test the network core bootloader sample run the following commands:
+   .. group-tab:: SEGGER Embedded Studio
 
-#. |connect_terminal|
+      Follow the instructions linked above, enabling the following options:
 
-   Note that on the nRF5340 DK has multiple UART instances, so the correct port must be identified.
+      * :option:`CONFIG_BOOTLOADER_MCUBOOT`:
 
-#. ``west build -b nrf5340dk_nrf5340_cpuapp -d build_netboot samples/bluetooth/peripheral_uart -f -- -DCONFIG_BOOTLOADER_MCUBOOT=y -Dmcuboot_CONFIG_PCD=y``
+        #. Select :guilabel:`Project` -> :guilabel:`Configure nRF Connect SDK project`.
+        #. Go to :guilabel:`Modules` -> :guilabel:`Nordic nRF Connect` -> :guilabel:`Bootloader` and set :guilabel:`Use MCUboot` to enable :option:`CONFIG_BOOTLOADER_MCUBOOT`.
+        #. Click :guilabel:`Configure`.
 
-   This sample (:file:`samples/bluetooth/peripheral_uart`) will automatically include the network core sample ``hci_rpmsg`` when built for the nRF5340 DK.
-   Since we enable MCUboot (``-DCONFIG_BOOTLOADER_MCUBOOT=y``) in the application core, the network core bootloader is automatically included as well.
-   In addition to this, it is necessary to enable the :ref:`subsys_pcd` subsys for the MCUboot image (``-Dmcuboot_CONFIG_PCD=y``).
+      * :option:`mcuboot_CONFIG_PCD`:
 
-#. ``west flash -d build_netboot``
+        #. Select :guilabel:`Project` -> :guilabel:`Configure nRF Connect SDK project`.
+        #. Go to :guilabel:`Modules` -> :guilabel:`Nordic nRF Connect` -> :guilabel:`Peripheral CPU DFU (PCD)` and set :guilabel:`Use Peripheral CPU DFU (PCD)` to enable :option:`mcuboot_CONFIG_PCD`.
+        #. Click :guilabel:`Configure`.
 
-   Program the merged hex files for both domains (application core and network core).
+      The :ref:`peripheral_uart` sample will automatically include the network core sample :ref:`bluetooth-hci-rpmsg-sample` when built for the nRF5340 DK.
+      Enabling MCUboot in the application core automatically includes the network core bootloader.
+      In addition to this, you must also enable the :ref:`subsys_pcd` library for the MCUboot image.
 
-#. ``nrfjprog --program build_netboot/zephyr/net_core_app_moved_test_update.hex --sectorerase -r``
+   .. group-tab:: Command Line
+
+      #. |connect_terminal|
+
+         Note that the nRF5340 DK has multiple UART instances, so the correct port must be identified.
+
```

Comment:
I recommend refactoring this step-by-step procedure by having a brief description as the bullet, followed by a code block of the command, then followed by its additional information.

* Steps are then succinctly described rather than described by dissecting the command.
* Aligns with most other bullet/number lists across our documentation.
* Allows these commands to make use of restructuredText's `code-block` or `parsed-literal` directives.

Reply:
fixed